### PR TITLE
Freeze libcalico requirements

### PR DESCRIPTION
**Pull request by tomdee**

### CODE REVIEW

File: build-requirements.txt [hunk 1]
```diff
@@ -1,9 +1,8 @@
-setuptools==0.9.8
-docopt==0.6.2
-PyInstaller==3.1
-six==1.9.0
-docker-py==1.7.0
-prettytable==0.7.2
-netaddr==0.7.15
-ConcurrentLogHandler==0.9.1
-pyyaml==3.11
+setuptools
```
Comment (caseydavenport) [inline build-requirements.txt:1]: Why are there so many fewer requirements in this file than in the frozen one?

Comment (tomdee) [inline build-requirements.txt:1]: Because the frozen one contains all the sub-requirements

Comment (caseydavenport) [inline build-requirements.txt:1]: Aha.. silly me.

**Comment by caseydavenport** [comment]
LGTM

**Comment by caseydavenport** [comment]
LGTM

**Pull request merged**